src/picomsg/src/oper_db_writer.cpp

```diff
@@ -59,3 +59,50 @@ bool oper_db_writer_terminator_state_change(const uint32_t terminator_id, const
 
 	return true;
 }
+
+bool oper_db_writer_set_terminator_tunnel_counters(const uint32_t terminator_id, const uint64_t count) {
+	auto oper_db_client = picomsg_get_oper_db_client();
+	std::string terminator_id_str = std::to_string(terminator_id);
+	const char* keys[] = { terminator_id_str.c_str()};
+	auto db_obj = oper_db_client_create_db_object(*oper_db_client, LIBOPER_DB_CLIENT_CLASS_ID_TERMINATOR, keys);
```

Comment:
Please add more spacings between the variables and other logic here, having a lot of variables bunched up like so makes it hard to read. (Ditto for `oper_db_writer_set_term_vrf_tunnel_counter()`)